### ISSUE TITLE
help with reporting issues

### ISSUE THREAD

**Issue by harperjf** [lfortran/lfortran#4057]
If I notice I have just made a mistake in something I send to Fortran Discourse I can correct it within a few minutes. Can I do that when reporting an lfortran issue (if so how?) or must I leave the wrong report as is and send a correction as a comment on it?

**Comment by certik** [comment]
Yes, just click on the three dots at the top right corner of your comment / issue and click "Edit". Then you can edit your comment / issue.

**Comment by HarshitaKalani** [comment]
Closing this.
